### PR TITLE
GDV-254 SVG images to frontend

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: components/graph-line.vue [hunk 1]
```diff
@@ -219,6 +231,12 @@ export default {
 </script>
 
 <style>
+#graphImage {
```
Comment (SiggyF) [inline components/graph-line.vue]: css  naming

File: store/map/index.js [hunk 2]
```diff
@@ -140,20 +117,53 @@ export const actions = {
...
+          let serie = []
+          const eventResults = response.results.filter(res =>
+            _.has(res, 'events'),
+          )
+
+          eventResults.forEach(res => {
+            serie = serie.concat(res.events.map(event => event.value))
+            category = category.concat(
+              res.events.map(event =>
+                moment(event.timeStamp).format('MM-DD-YYYY HH:mm'),
```
Comment (SiggyF) [inline store/map/index.js]: what's with the , at the end of the line?

File: store/map/index.js [hunk 3]
```diff
@@ -140,20 +117,53 @@ export const actions = {
...
-      return getFromApi('timeseries', parameters).then(({ results }) => {
-        commit(
-          'datasets/addDatasetPointData',
-          Object.freeze({
-            id: datasetId,
-            data: {
-              [locationId]: {
-                category: getFormattedTimeStamps(results),
-                serie: getValues(results),
+        if (pointDataType === 'images') {
```
Comment (SiggyF) [inline store/map/index.js]: Please add documentation.

File: store/map/index.js [hunk 4]
```diff
@@ -140,20 +117,53 @@ export const actions = {
...
+            category = category.concat(
+              res.events.map(event =>
+                moment(event.timeStamp).format('MM-DD-YYYY HH:mm'),
+              ),
+            )
+          })
+
+          commit(
+            'datasets/addDatasetPointData',
+            Object.freeze({
```
Comment (SiggyF) [inline store/map/index.js]: Why freeze?

File: test/unit/store/map/index/actions.spec.js [hunk 5]
```diff
@@ -126,7 +126,24 @@ describe('storeActiveDatasets', () => {
 })
 
 describe('loadPointDataForLocation', () => {
-  test('loads point data for the specified location', async () => {
+  const state = {
+    datasets: {
+      ab: {},
```
Comment (SiggyF) [inline test/unit/store/map/index/actions.spec.js]: please use more meaningful properties. This  is a bit vague.

File: test/unit/store/map/index/actions.spec.js [hunk 6]
```diff
@@ -187,29 +249,29 @@ describe('loadPointDataForLocation', () => {
...
         .add(5, 'days')
         .format('YYYY-MM-DDTHH:mm:ssZ'),
-      locationId: 'ef',
+      locationId: 'loc1',
       startTime: moment()
         .subtract(3, 'days')
         .format('YYYY-MM-DDTHH:mm:ssZ'),
     })
-    expect(commit.mock.calls[0]).toEqual([
+    expect(commit.mock.calls[1]).toEqual([
```
Comment (SiggyF) [inline test/unit/store/map/index/actions.spec.js]: I find this test very complex. Can  you document it?

File: store/map/index.js (hunk 2 above)
Comment (CindyvdVries) [reply store/map/index.js]: linter settings..

File: store/map/index.js (hunk 3 above)
Comment (CindyvdVries) [reply store/map/index.js]: check

File: store/map/index.js (hunk 4 above)
Comment (CindyvdVries) [reply store/map/index.js]: Good point

File: test/unit/store/map/index/actions.spec.js (hunk 6 above)
Comment (CindyvdVries) [reply test/unit/store/map/index/actions.spec.js]: CHanged it. I did two requests for different parameters, but the first request was checked in previous test, second request needed to be checked in this test.. But I shouldn't have send that irst request again in this test. Fixed that

File: test/unit/store/map/index/actions.spec.js (hunk 5 above)
Comment (CindyvdVries) [reply test/unit/store/map/index/actions.spec.js]: Changed it to par1, par2, par3 :)

**Pull request merged**